src/lib/y2storage/blk_device.rb

```diff
@@ -317,6 +317,12 @@ def delete_filesystem
       remove_descendants
     end
 
+    def mount_point
```

Comment:
blk_device is so important that it really deserves documentation for any of its methods.